packages/react-text/tsconfig.json

```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "target": "ES5",
     "module": "CommonJS",
-    "lib": ["es5", "dom"],
+    "lib": ["es2015", "dom"],
```

Comment:
you run into make-styles issue as well ? (getting ts errors)

Reply:
Yep, with `mergeClasses` and `createCSSVariablesProxy`.

Reply:
kk sounds good